app/src/components/main/analytics/events/filtersPageEvents.js

```diff
@@ -36,11 +39,17 @@ export const getAddEditFilterModalEvents = (isEditMode) => {
       action: `Click on button Cancel in Modal ${modalType} Filter`,
       label: `Close Modal ${modalType} Filter`,
     },
-    clickOkBtn: {
-      category: FILTERS_PAGE,
-      action: `Click on button ${actionType} in Modal ${modalType} Filter`,
-      label: `${actionType} filter in Modal ${modalType} Filter`,
-    },
+    clickOkBtn: isEditMode
+      ? {
```

Comment:
Why do we need to use a ternary operator here?
And FYI : if the event does not contain a place parameter, that event does not send to GA-4